cypress/objects/loginPage.js

```diff
@@ -0,0 +1,26 @@
+class LoginPage {   //klasiu pav is didziosios
+    visit() {
+        cy.visit('/');  //homepage
+    }
+    getValidationError() {
+        return cy.get('.Select.not-valid');
+    }
+    getSubmit() {
```

Comment:
You could add "button" to the end, this way it's always clearer what kind of element we're interacting with, in this case a button :)

`getSubmitButton()`